pygmt/src/colorbar.py

```diff
@@ -49,11 +49,14 @@ def colorbar(self, **kwargs):
         Set color bar boundary frame, labels, and axes attributes.
     {CPT}
     position : str
-        ``[g|j|J|n|x]refpoint[+wlength[/width]][+e[b|f][length]][+h|v]
-        [+jjustify][+m[a|c|l|u]][+n[txt]][+odx[/dy]]``. Defines the
-        reference point on the map for the color scale using one of four
-        coordinate systems: (1) Use *g* for map (user) coordinates, (2) use
-        *j* or *J* for setting refpoint via a 2-char justification code
+        [**g**\|\ **j**\|\ **J**\|\ **n**\|\ **x**]\ *refpoint*
+        [**+w**\ *length*\ [/\ *width*]]\ [**+e**\ [**b**\|\ **f**][*length*]]
+        [**+h**\|\ **v**][**+j**\ *justify*]
+        [**+m**\ [**a**\|\ **c**\|\ **l**\|\ **u**]]
+        [**+n**\ [*txt*]][**+o**\ *dx*\ [/*dy*]].
+        Defines the reference point on the map for the color scale using one of
+        four coordinate systems: (1) Use *g* for map (user) coordinates, (2)
+        use *j* or *J* for setting refpoint via a 2-char justification code
```

Comment:
Following https://github.com/GenericMappingTools/pygmt/blob/master/CONTRIBUTING.md#example-code-standards, literal GMT arguments should be bold, 
```suggestion
        Defines the reference point on the map for the color scale using one of
        four coordinate systems: (1) Use **g** for map (user) coordinates, (2)
        use **j** or **J** for setting *refpoint* via a 2-char justification code
```